packages/create-block/lib/init-block-json.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+const { isEmpty, omitBy } = require( 'lodash' );
+const { join } = require( 'path' );
+const { writeFile } = require( 'fs' ).promises;
+
+/**
+ * Internal dependencies
+ */
+const { info } = require( './log' );
+
+module.exports = async (
+	blockTemplate,
+	{ slug, namespace, title, description, category, dashicon, textdomain }
+) => {
+	const outputFile = join( process.cwd(), slug, 'block.json' );
+	const template = blockTemplate.defaultValues.slug.split( '-' ).shift();
```

Comment:
This check isn’t reliable in the long run, it’s very likely that a slug without the template name in it is going to be provided in the future.

Instead, we can put both `editorStyle` and `editorScript` inside the template object. Sn override for `es5` can be added here:

https://github.com/WordPress/gutenberg/blob/66e5e5b20c759845866aa172c6a77bdd287ce7b1/packages/create-block/lib/templates.js#L22

The default value can be added here:

https://github.com/WordPress/gutenberg/blob/66e5e5b20c759845866aa172c6a77bdd287ce7b1/packages/create-block/lib/templates.js#L81

The value should be prefixed with `file:`. See more details in block registration RFC:

https://github.com/WordPress/gutenberg/blob/master/docs/rfc/block-registration.md#assets

Reply:
> This check isn’t reliable in the long run

Good point. Updated here: 7d4a923f3